edx-olive/edx-platform
# [SE-3253] Fix translations for edx-platform

This PR adds the translation strings marked as Priority 1 from the spreadsheet in edx-platform code.

**JIRA tickets:** [SE-3253](https://tasks.opencraft.com/browse/SE-3253), [CAM2-353](https://lnet-jira.atlassian.net/browse/CAM2-353)

**Testing Instructions:**
Changes have been deployed to stage. There is a course called "Test Course" that contains some of the components specified.
- Go over items in spreadsheet that are marked as Priority 1.
- Create required components and verify the translations by selecting hebrew lang.


**Author's notes and concerns:**
- This PR only includes the changes to the edx-platform, it should be tested along with other repo PRs included in the task


**Reviewers:**
